modules/representation_management/lib/representation_management/v0/pdf_constructor/form_2122.rb

```diff
@@ -185,6 +170,48 @@ def page2_options(data)
             "#{PAGE2_KEY}.I_Authorize[0]": data.consent_address_change == true ? 1 : 0
           }
         end
+
+        private
+
+        def add_representative_contact(pdf, representative, organization_name)
+          rep_name = format_name(representative)
+          add_text_with_spacing(pdf, rep_name, style: :bold, move_down: 8)
+          pdf.font('soursesanspro') do
+            pdf.text(organization_name)
+            add_address(pdf, representative)
+            pdf.move_down(5)
+            pdf.text(format_phone_number(representative.phone))
```

Comment:
This is something not documented well: the legacy `Veteran::Service::Representative` has both `phone` and `phone_number`. 
- `phone` gets updated by the nightly excel file job in `modules/veteran/app/sidekiq/veteran/vso_reloader.rb` but it uses the organization phone, not the representative phone.
- `phone_number` gets updated from the Trexler File in `modules/veteran/app/sidekiq/representatives/update.rb` and uses the individual's work phone number according to OGC.

We probably want `phone_number`. Unfortunately, the new `AccreditedIndividual`'s attribute is `phone` so something will have to change to make the code work for both models.

Reply:
I put the conditional logic to get `phone_number` from the `Veteran::Service::Representative` in the data model: `modules/representation_management/app/models/representation_management/form_2122_base.rb`.  I also moved the logic for `individual_type ` there so I won't need a review from lighthouse.